### PR TITLE
update webpack & config

### PR DESCRIPTION
**Pull request by umurb**

### CODE REVIEW

File: app/containers/LoginHome/index.js [hunk 1]
```diff
@@ -257,49 +286,81 @@ class LoginHome extends Component<Props> {
     history.push(`${match.path}/${route}`);
   };
 
+  onLedgerConnect = () => {
```
[the remark this answers is not in the document]
Comment (developer0623) [reply app/containers/LoginHome/index.js]: @pat310 I'm working with the functions, now

File: app/containers/LoginHome/index.js [hunk 2]
```diff
@@ -257,49 +286,81 @@ class LoginHome extends Component<Props> {
...
+                    </Trans>
+                  </div>
+                </Tip>
+              </CardContainer>
+              <CardContainer>
+                <CardImg src={ledgerImg} />
+                <CardTitle>{t('containers.loginHome.ledger_wallet')}</CardTitle>
+                <CreateWalletButton
+                  buttonTheme="primary"
+                  onClick={() => this.onLedgerConnect()}
```
[the remark this answers is not in the document]
Comment (developer0623) [reply app/containers/LoginHome/index.js]: Good suggestion.
After confirmed the params, I'll fix those, now

**Comment by umurb** [comment]
It seems like this PR is attempting to change the LICENSE from GPL to MIT. We shouldn't do that

File: app/package.json [hunk 3]
```diff
@@ -11,9 +11,11 @@
   },
   "scripts": {
     "electron-rebuild": "node -r babel-register ../internals/scripts/ElectronRebuild.js",
-    "postinstall": "npm run electron-rebuild"
+    "postinstall": "yarn electron-rebuild"
   },
+  "license": "MIT",
```
Comment (umurb) [inline app/package.json]: Can we replace this line to;
`"license": "GPL-3.0-only",`

or just delete it if not necessary?

**Pull request merged**